quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -186,21 +164,13 @@ impl MappingLeaf {
 
     fn populate_json<'a>(
         &'a self,
-        named_doc: &mut BTreeMap<String, Vec<JsonValue>>,
+        named_doc: &mut BTreeMap<String, Vec<Value>>,
```

Comment:
```suggestion
        named_doc: &mut BTreeMap<String, Vec<TantivyValue>>,
```

Reply:
@guilload 
My initial understanding of this when I worked on the search result snippet structure was that we wanted to make the distinction for JsonValue when we have conflicting Value types in a file. 
Now I have the feeling that we want to always have the distinction between JsonValue and TantivyValue clear. In the later case, I think a separate PR should be better since we use these value types in many places.
I have fixed both files I touched on here but let me know if we should do the refactor across the whole project. I will create an issue.

Reply:
Yes, I'd rather always distinguish them. You don't always get to see the imports when reviewing on GitHub, and as the size of the codebase increase, I have a hard time keeping track of what's what.